twitter-producer/build.gradle

```diff
@@ -1,12 +1,18 @@
+import com.github.davidmc24.gradle.plugin.avro.GenerateAvroJavaTask
+
 plugins {
     id 'java'
     id 'application'
+    id "com.github.davidmc24.gradle.plugin.avro-base" version "1.2.0"
 }
 
 group 'com.kafkatweets'
 version '0.1'
 
 repositories {
+    maven {
+        url 'http://packages.confluent.io/maven/'
```

Comment:
@daveklein this should allow me to use io.confluent.kafka.serializers.KafkaAvroSerializer but it's not working